src/datetime.jl

```diff
@@ -77,3 +79,11 @@ end
 function ==(a::MySQLDateTime, b::MySQLDateTime)
     a.date == b.date && a.time == b.time
 end
+
+@require Dates begin
+using Dates
```

Comment:
Dates is not that big a dependency, and it is built into base in 0.4. Is Requires.jl worth it here?